services/consensusalgo/benchmarkconsensus/common.go

```diff
@@ -41,7 +41,7 @@ func (s *service) setLastCommittedBlock(blockPair *protocol.BlockPairContainer,
 }
 
 func (s *service) requiredQuorumSize() int {
-	return int(math.Ceil(float64(s.config.NetworkSize(0)) * float64(s.config.ConsensusContextPercentageOfNodesRequiredForConsensus()/100)))
+	return int(math.Ceil(float64(s.config.NetworkSize(0)) * float64(s.config.ConsensusRequiredQuorumPercentage()/100)))
```

Comment:
why not make the config a float?

Reply:
Because Kirill doesn't support float config values and didn't want to add them just for a single field